src/routes/safe/store/actions/createTransaction.js

```diff
@@ -94,6 +99,23 @@ const createTransaction = ({
   }
 
   try {
+    if (!isExecution && !isSmartContractWallet && semverSatisfies(safeVersion, '>=1.1.1')) {
```

Comment:
so theoretically this works with all versions if we use EIP712, but I think it is ok to only allow it for the latest contracts (maybe add a comment)